apps/ledger-live-mobile/e2e/specs/deeplinks.spec.ts

```diff
@@ -4,7 +4,7 @@ import { getElementByText, isAndroid } from "../helpers";
 import AccountPage from "../models/accounts/accountPage";
 import AccountsPage from "../models/accounts/accountsPage";
 import CustomLockscreenPage from "../models/stax/customLockscreenPage";
-import DiscoveryPage from "../models/discovery/discoveryPage";
+import DiscoveryPage from "../models/discover/discoverPage";
```

Comment:
Just changing to `Discover` instead of `Discovery` to match with page title and component names